libs/stats/odc/stats/proc.py

```diff
@@ -89,17 +92,17 @@ def prep_stage(tasks: Iterable[Task],
 
         if isinstance(ds, xr.DataArray):
             attrs = ds.attrs.copy()
-            ds = ds.to_dataset(dim='band')
+            ds = ds.to_dataset(dim="band")
             for dv in ds.data_vars.values():
                 dv.attrs.update(attrs)
 
-        cog = client.compute(sink.dump(task, ds),
-                             fifo_timeout='1ms')
+        cog = client.compute(sink.dump(task, ds), fifo_timeout="1ms")
         rr = dask_wait(ds)
         assert len(rr.not_done) == 0
         del ds, rr
         in_flight_cogs.add(cog)
 
     done, _ = drain(in_flight_cogs)
     for r in done:
-        yield r
+        print(r)
```

Comment:
Is this mean to be here?

Reply:
Yeah, for now - I'm using it for debugging.